src/main/asciidoc/inc/misc/_env.adoc

```diff
@@ -1,6 +1,22 @@
 
 
-When creating a container one or more environment variables can be set via configuration with the `env` parameter
+When creating a container one or more environment variables can be set via configuration with the `env` parameter. Somehow maven is showing weird behavior in case of interpolating single ${..} parameter; so we have provided a workaround for that. An `env` can take the following forms:
+
+.Env format
```

Comment:
Thanks for caring about the documentation, too ;-)

However I would describe it a bit shorter. Maybe just: 

> If the only value of the env parameter is a docker-maven-plugin internal property which has been set implicitely you have to prefix the property with a single `+` like in `+${docker.container.test.ip}`. This is necessary due to some Maven limitations which simply interpolates a lone, non defined property, to an empty string which can't then be replaced by this plugin after the initial interpolation phase.

Maybe we can list the affected properties here. Or we should put the whole documentation to the place, where the implicit properties are explained (i think this is even better)